dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/support/RpcUtils.java

```diff
@@ -183,14 +182,6 @@ public static boolean isAsync(URL url, Invocation inv) {
         return isAsync;
     }
 
-    public static boolean isGeneratedFuture(Invocation inv) {
-        return Boolean.TRUE.toString().equals(inv.getAttachment(Constants.FUTURE_GENERATED_KEY));
-    }
-
-    public static boolean hasGeneratedFuture(Method method) {
-        return method.isAnnotationPresent(AsyncFor.class) && hasFutureReturnType(method);
-    }
-
     public static boolean isFutureReturnType(Invocation inv) {

```

Comment:
Just an thought should we consider this name to ** isReturnTypeFuture** instead of isFutureReturnType

Reply:
done.